api/mesh/v1alpha1/meshgateway/rest.yaml

```diff
@@ -0,0 +1,115 @@
+openapi: 3.0.3
+info:
+  version: v1alpha1
+  title: Kuma API
+  description: Kuma API
+  x-ref-schema-name: "MeshGateway"
+
+paths:
+  /meshgateways/{name}:
```

Comment:
This is missing `/meshes/{meshName}` rigth?

Reply:
yes, scope change fixed that